src/zenml/cli/utils.py

```diff
@@ -391,6 +391,41 @@ def format_integration_list(
     return list_of_dicts
 
 
+def print_stack_outputs(stack: "StackResponseModel") -> None:
+    """Prints outputs for stacks deployed with mlstacks.
+
+    Args:
+        stack: Instance of a stack model.
+    """
+    if not stack.stack_spec_path:
+        declare("No stack spec path is set for this stack.")
+        return
+    stack_caption = f"'{stack.name}' stack"
+    rich_table = table.Table(
+        box=box.HEAVY_EDGE,
+        title="MLStacks Outputs",
+        caption=stack_caption,
+        show_lines=True,
+    )
+    rich_table.add_column("OUTPUT_KEY", overflow="fold")
+    rich_table.add_column("OUTPUT_VALUE", overflow="fold")
+
+    from mlstacks.utils.terraform_utils import get_stack_outputs
```

Comment:
`mlstacks` is a core dependency now so this will never fail right?

Reply:
Oh you're totally right I should add a check for that. Thank you! (`mlstacks` is not a core dependency just as soon with #1611 `python-terraform` won't be either.)

Reply:
Added the check for mlstacks installation now.